client/components/seo-preview-pane/index.jsx

```diff
@@ -177,15 +178,19 @@ export class SeoPreviewPane extends PureComponent {
 	constructor( props ) {
 		super( props );
 
+		const defaultService = props.post ? 'wordpress' : 'google';
 		this.state = {
-			selectedService: props.post ? 'wordpress' : 'google'
+			selectedService: defaultService
 		};
 
 		this.selectPreview = this.selectPreview.bind( this );
+
+		recordTracksEvent( 'calypso_seo_tools_social_preview', defaultService );
```

Comment:
warning. this won't actually do anything. the `recordTracksEvent()` function here is just an action creator. it will return the action that should be dispatched into Redux.

it needs to be passed in to the props via `mapDispatchToProps` something like this…

```js
const { trackPreviewService } = this.props;

trackPreviewService( defaultService );

const mapDispatchToProps = dispatch => ( {
	trackPreviewService: service => dispatch( recordTracksEvent( 'calypso_seo_tools_social_preview', service ) ),
} );
```